lib/plugins/console/generate.js

```diff
@@ -154,13 +157,25 @@ function generateConsole(args) {
   });
 }
 
+function wrapDataStream(dataStream, options) {
+  var ignoreErrors = options && options.ignoreErrors
+
+  if (ignoreErrors) {
+    var noop = () => {}
+    dataStream.on('error', noop)
+    return dataStream.pipe(new PassThrough())
```

Comment:
`PassThrough` will pass data ignoring errors and this newly added `.on('error', reject)` will never be called in this case.